Sources/AccountSDKIOSWeb/Simplified Login/View/SimplifiedLoginNavigationController.swift

```diff
@@ -1,9 +1,18 @@
 import UIKit
 
 class SimplifiedLoginNavigationController: UINavigationController {
-
+    static let isPad: Bool = UIDevice.current.userInterfaceIdiom == .pad
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.backgroundColor = .clear
+        
+        if Self.isPad {
+            modalPresentationStyle = .formSheet
+            preferredContentSize = .init(width: 450, height: 424)
+        } else {
+            modalPresentationStyle = .overFullScreen
+        }
```

Comment:
Wasn't this logic setting the SimplifiedLoginViewController before? This change, is applying it to the SimplifiedLoginNavigationController

Reply:
No, it never was in SimplifiedLoginViewController. I replied to your first comment that this should go to navigation controller.